main.go

```diff
@@ -312,10 +368,10 @@ func main() {
 		}
 
 		res, err := k.HandleCard(ev.UID)
-		if err != nil {
-			fmt.Fprintln(os.Stderr, err)
+		if res != nil {
+			res.Print(lcd)
 		} else {
-			fmt.Println(res)
+			flashLCD(lcd, err.Error(), 255, 0, 0)
```

Comment:
I agree, but in my eyes, we should filter and parse them in some way before printing to a 16(or 32, if you wish)-character-display without working line-breaks ;)